packages/react-native/src/babel-prefix-resolve.ts

```diff
@@ -0,0 +1,9 @@
+export function babelPrefixResolve(dep: string, prefixType?: string): string {
+  return dep.startsWith('module:')
+    ? dep.substring(7)
+    : dep.startsWith('@babel/')
+    ? dep
+    : prefixType && !dep.startsWith(`babel-${prefixType}`)
+    ? `babel-${prefixType}-${dep}`
+    : dep;
+}
```

Comment:
I'll move this function and tests to be under common folder, so it will be better when we will need this for the new Babel compiler for example.